sdk_container/src/third_party/coreos-overlay/profiles/coreos/targets/generic/dev/profile.bashrc

```diff
@@ -0,0 +1,29 @@
+# Adapted from cross-boss. Only needed for the dev profile because we don't
+# build the SDK with sysroot flags and we don't install build tools to the
+# production image.
+cros_pre_pkg_preinst_strip_sysroot() {
+	local FILES FILE
+
+	# Gather all the non-binary files with the --sysroot or --with-sysroot flag.
+	mapfile -t -d '' < <(find "${D}" -type f -exec grep -lZEIe "--sysroot=${ROOT}\b|--with-sysroot=${EROOT}\b" {} +) FILES
```

Comment:
I'm uneasy seeing some variables passed to `grep` as a regexp. I hope we won't see some weird paths in `${ROOT}` or `${EROOT}`. Maybe we could escape those paths somehow? Ideally, we could pass `-F` instead of `-E` to `grep`, but then `\b` won't work…

Reply:
I'm also curious if there are files that quote (either with single quotes or double quotes) the value of the flag (`--sysroot="/quoted/path"`).

Reply:
I'll see whether I can use fixed strings, although these values are only ever going to be things like `/build/arm64-usr`.

We control the `--sysroot` flag, so it will never have quotes. Similarly, the `--with-sysroot` flag comes from Portage itself.

Reply:
I think you could use fixed strings. There is a "risk" that this would pick up more files (with `--sysroot=/build/amd64-usr/foo` out of the blue), but your sed invocation should be careful enough to ignore the non-matching lines.